cfme/tests/automate/test_automate_manual.py

```diff
@@ -767,3 +767,268 @@ def test_variable_pass_method_playbook():
+               'name' attribute equals Domain_Test02
+               NOTE: Do not change the 'source' attribute from 'system' to 'user' this time.
+            4. Archived the new Domain_Test02 dir and upload to CloudForms Appliance
+        expectedResults:
+            1.
+            2.
+            3.
+            4. The imported domains should be Custom Domains or these imported domains should gets
+               deleted from db
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(coverage=[1753523])
+def test_attribute_value_message():
+    """
+    Bugzilla:
+        1753523
+
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseposneg: positive
+        casecomponent: Automate
+        testSteps:
+            1. Create domain, namespace, class and instance pointing to method
+            2. Navigate to automate > automation > simulation page
+            3. Fill values for attribute/value pairs of namespace, class, instance and add message
+               attribute with any value and click on submit.
+            4. See automation.logs
+        expectedResults:
+            1.
+            2.
+            3.
+            4. Custom message attribute should be considered with instance in logs
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(coverage=[1752875])
+def test_existing_domain_child_override():
+    """
+    PR:
+     https://github.com/ManageIQ/manageiq-ui-classic/pull/4912
+
+    Bugzilla:
+        1752875
+
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseposneg: positive
+        casecomponent: Automate
+        testSteps:
+            1. Create custom domain and copy class - "ManageIQ/System/Request"
+            2. Lock this domain
+            3. Navigate to Automation > automate > Import/export and click on "export all classes
+               and instances to file"
+            4. Go to custom domain and unlock it. Remove instance - "ManageIQ/System/Request/" and
+               copy - "ManageIQ/System/Process/Request" (you can copy more classes or methods or
+               instances) to custom domain and again lock the domain.
+            5. Go to import/export page and click on 'choose file'. Select exported file and click
+               on upload
+            6. Select "Select domain you wish to import from:" - "custom_domain" and check Toggle
+               All/None
+            7. Click on commit button. You will see flash message: "Error: Selected domain is
+               locked"
+            8. Then navigate to custom domain and unlock it
+            9. Perform step 5, 6 and 7(In this case, domain will get imported)
+            10. Go to custom domain
+        expectedResults:
+            1.
+            2.
+            3.
+            4. You should see existing as well as imported namespaces, classes, instances or methods
```

Comment:
Afaik I believe you need `expectedResults` 5-10 in order for this metadata to be parsed correctly.